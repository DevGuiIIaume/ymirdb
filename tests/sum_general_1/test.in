set a 100
set b a 200
set c b 300
set d 500 600 700
sum a
sum b
sum c
sum d
bye
