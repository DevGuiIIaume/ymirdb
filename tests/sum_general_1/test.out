> ok

> ok

> ok

> ok

> 100

> 300

> 600

> 1800

> bye
